machine_learning/logistic_regression.py

```diff
@@ -55,7 +61,14 @@ def logistic_reg(
         J = cost_function(h, y)
 
         iterations += 1  # update iterations
-
+        
+     for step in range(num_steps):
+          scores = np.dot(X, weights)
+          predictions = sigmoid(scores)
+          if step % 10000 == 0:
+               print log_likelihood(X,Y,weights)     # Print log-likelihood every so often
```

Comment:
__print()__ is a function in Python 3.  --> __print(log_likelihood(X,Y,weights))__